src/schema/v2/me/index.ts

```diff
@@ -578,6 +578,15 @@ const MeField: GraphQLFieldConfig<void, ResolverContext> = {
       "showsByFollowedArtists",
       "newWorksFromGalleriesYouFollowConnection",
       "myCollectionInfo",
+      "watchedLotConnection",
+      "userInterestsConnection",
+      "recentlyViewedArtworksConnection",
+      "myCollectionConnection",
+      "artworkInquiriesConnection",
+      "collectionsConnection",
+      "auctionResultsByFollowedArtists",
+      "myCollectionAuctionResults",
+      "newWorksByInterestingArtists",
```

Comment:
Can we alphabetize this whole array?

Reply:
As far as I know, this occurs in a couple of places in metaphysics, and it could be resolved using the general ESLint rule `sort-keys`, perhaps. But definitely, as a code bestpractice/guideline improvement in separate PR, what do you think?